src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,15 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  comments.length
+    ? comments.map(comment => (
+      <div className="CommentList">
+        <CommentInfo
+          key={comment.id}
+          comment={comment}
+        />
+      </div>
+    ))
```

Comment:
Key should be on main tag or component
```suggestion
    ? comments.map(comment => (
      <div className="CommentList"  key={comment.id}>
        <CommentInfo
          comment={comment}
        />
      </div>
    ))
```

Reply:
Thanks!